src/anomalib/data/base/datamodule.py

```diff
@@ -42,6 +44,10 @@ def collate_fn(batch: list) -> dict[str, Any]:
         if "boxes" in elem:
             # collate boxes as list
             out_dict["boxes"] = [item.pop("boxes") for item in batch]
+        if "masks" in elem:
```

Comment:
Minor comment, but would it be an idea to rename `elem` to `sample`

Reply:
We use `elem` to match the naming in the default `collate` function from torch. Let's leave it like this, it's outside of the scope of this PR anyway.

Reply:
alright, let's leave it as it not-related to this PR, but poor choice by torch